tracer/src/Datadog.Trace/Span.cs

```diff
@@ -162,7 +162,7 @@ internal ISpan SetTag(string key, string value)
         {
             if (IsFinished)
             {
-                Log.Warning("SetTag should not be called after the span was closed");
+                Log.Warning("SetTag should not be called after the span was closed.\r\n{stack}", Environment.StackTrace);
```

Comment:
Do we definitely want to include the stack here? I know it's an edge case, but if they are calling this incorrectly, it's a programming error, which means it could be repeated a _lot_ 

If anything, we should probably log the `key` and `value` instead of the stack trace? Probably just as easy to find in their code base and much lighter on their app/logs?

Reply:
Oh, no you're right, this was a thing I added while debugging... I'll revert this change.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3276/commits/4837b4b60b2f11c07439c7b9c6faea26ca5e42ea